tests/gpts/github/test-config/github_positive_suite.ts

```diff
@@ -184,7 +183,17 @@ export const gitHubBasicGoldenPathTemplateTests = (gptTemplate: string) => {
                 }
             expect(image_index).not.toBe(undefined)
             } 
-        }, 900000)        
+        }, 900000)   
+        
+        /**
+         * verify if the ACS Scan is successfully done from the logs of task steps
+         */
+        it(`Check if ACS Scan is successful for ${gptTemplate}`, async ()=> {
+            const result = await checkIfAcsScanIsPass(repositoryName, developmentNamespace)
```

Comment:
I am curious why you don't check the task directly, for example the status of ACS scan task